README.rst

```diff
@@ -124,9 +128,69 @@ To update page objects installed from external repos:
 
     paver install_pages
 
+How to run Whitelabel tests?
+----------------------------
+
+Make sure you are in the correct folder.
+
+.. code:: bash
+
+    cd $HOME/edx-e2e-tests
+
+Before running whitelabel tests please ensure that these environment variables are set.
+
+.. code:: bash
+
+    ==> BASIC_AUTH_USER
+    ==> BASIC_AUTH_PASSWORD
+    ==> USER_LOGIN_EMAIL
+    ==> USER_LOGIN_PASSWORD
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
+    ==> GLOBAL_PASSWORD
+    ==> STAFF_USER_EMAIL
+    ==> TEST_EMAIL_SERVICE
+    ==> TEST_EMAIL_ACCOUNT
+    ==> TEST_EMAIL_PASSWORD
+    ==> ACCESS_TOKEN
+
+
+To run all the tests in the file:
+
+.. code:: bash
+
+    paver e2e_test whitelabel/test_user_account.py
+
+To run all the tests in a particular class:
```

Comment:
I think we can remove the running specific classes/tests parts from the white label section (since it is the same as the e2e section, except for the paver target. Instead, you could just make a note about using the same Nose test specifiers as mentioned above, adding that you must use the white label paver target. What do you think?